dfsclient/src/main/java/org/hpccsystems/dfs/client/FileUtility.java

```diff
@@ -1095,7 +1149,8 @@ private static void performRead(String[] args, TaskContext context)
         for (int i = 0; i < datasets.length; i++)
         {
             String datasetName = datasets[i];
-            context.startOperation("Read " + datasetName);
+            context.startOperation("FileUtility.Read_" + datasetName);
+            context.setTaskSpanAttributes(Attributes.of(AttributeKey.stringKey("server.url"), connString));

```

Comment:
is there a way to add individual attributes after this initial set attributes call?